Examples/ReloadDataExample/ReloadDataViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ReloadDataViewController.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-04.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import UIKit
+import CollectionKit
+
+let maxDataCount: Int = 1000
+
+class ReloadDataViewController: CollectionViewController {
+  var data: [Int] = []
+  var dataSet: Set<Int> = Set()
```

Comment:
let see if we can make this simpler and remove the `dataSet` variable. From what I can see it is only used to generate the data which is not something we want to show in the example.